deny.toml

```diff
@@ -58,6 +58,7 @@ allow = [
     "LicenseRef-ring",
     "MIT",
     "MPL-2.0",
+    "NCSA",
```

Comment:
This should also be included in `about.toml` otherwise it'll fail while doing a release

Reply:
ah. okay!

Reply:
deaa91bf1